src/app/docs/action-hub/action-hub-docs.component.ts

```diff
@@ -1,14 +1,12 @@
 import { Component } from '@angular/core';
 
-import { SkyActionHubData } from '../../public/modules/action-hub/types/action-hub-data';
-
 @Component({
   selector: 'app-action-hub-docs',
   styleUrls: ['action-hub-docs.component.scss'],
   templateUrl: 'action-hub-docs.component.html'
 })
 export class ActionHubDocsComponent {
-  public data: SkyActionHubData = {
+  public data = {
     title: 'Action Hub',
```

Comment:
I think having the button "+ New" would be good, it is a common use case and would help reinforce the change in language from Add to New. I don't know if we even need the button to do anything, but wiring it up to a SKYUX modal is fine if we want to show that.

For the needs attention items, I don't think we need to capitalize the referenced functionality.

For the links, I think it's ok if they just point to "#" or whatever keeps the user where they are. Everyone should understand what a hyperlink is supposed to do in a real product.

Reply:
@blackbaud-johnly @Blackbaud-ToddRoberts I've updated the docs page demo so that buttons and links show a Sky UX modal (buttons previously used a native browser alert).

https://developer.blackbaud.com/skyux-pages?_sp.spa=pr-create-docs-and-demo

Let me know if that helps.

Reply:
The modals and other changes look great, but I have a few questions for Todd on the new buttons. I pinged him on Slack so that we don't spin our wheels in GitHub comments. I'll circle back after I get his feedback...